src/sentry/similarity/index.py

```diff
@@ -47,7 +47,7 @@ def __build_signatures(self, items):
             for i, bucket in enumerate(bands):
                 data[idx][i][bucket] += 1
 
-        arguments = [len(data)]
+        arguments = []
```

Comment:
This isn't needed since this is always passed as a variadic argument. (It will likely be changing in the future anyway.)